src/phoenix/trace/schemas.py

```diff
@@ -11,7 +11,6 @@ class SpanStatusCode(Enum):
     ERROR = "ERROR"
 
 
-@dataclass(frozen=True)
```

Comment:
Is this being removed because it's redundant to have an enum as a dataclass as well?